_components/header/header.html

```diff
@@ -92,85 +86,52 @@ <h2>Header usability</h2>
   </li>
   <li>
-    Use lists for your nav links — doing so enables screen readers to decipher
-    header contents.
+    <strong>Use list elements for your navigation links.</strong> This allows screen readers users to more easily
+    navigate header content.
   </li>
   <li>
-    If you’re using a logo that’s an image rather than text, make sure you
-    include alternative text for screen readers.
+    <strong>Add alt text to your logo image.</strong> If you’re using a logo that’s an image rather than text, make sure
+    you include alternative text for screen readers.
   </li>
   <li>
-    If you’re using a logo that’s text, use an <code>em</code>, not an
+    <strong>Don't use an H1 for your logo.</strong> If you’re using a logo that’s text, use an <code>em</code>, not an
     <code>h1</code>, unless it’s the homepage. Find more information here:
-    <a href="http://csswizardry.com/2010/10/your-logo-is-an-image-not-a-h1/"
-      >http://csswizardry.com/2010/10/your-logo-is-an-image-not-a-h1/.</a
-    >
+    <a
+      href="http://csswizardry.com/2010/10/your-logo-is-an-image-not-a-h1/">http://csswizardry.com/2010/10/your-logo-is-an-image-not-a-h1/.</a>
   </li>
 </ul>
 
-<h2>The “official government website” banner</h2>
-<p>
-  We recommend that all federal government websites include the “official
-  government website” banner and a logo or site name. The banner clearly
-  identifies your website as part of the United States government, while a logo or
-  site name helps visitors understand who you are and what you do.
-</p>
-<h3>Building with HTTPS</h3>
-<p>
-  <a
-    href="https://18f.gsa.gov/2014/11/13/why-we-use-https-in-every-gov-website-we-make/"
-    >HTTPS is a best practice</a
-  >
-  for all websites: It’s secure, private, and fast, and search engines prefer
-  encrypted sites. But it’s especially important for government websites. The
-  .gov in government websites carries a lot of weight. The public expects the
-  contents of .gov websites to be official and accurate, and they expect any
-  information they submit to that website to be private. The U.S. CIO provides
-  guidance to implementing HTTPS for government websites at
-  <a href="https://https.cio.gov/">https://https.cio.gov/.</a>
-</p>
-
 <h2>Further reading</h2>
 <ul>
   <li>
-    <a href="https://www.nngroup.com/articles/menu-design/"
-      >https://www.nngroup.com/articles/menu-design/</a
-    >
+    <a href="https://www.nngroup.com/articles/menu-design/">https://www.nngroup.com/articles/menu-design/</a>
   </li>
   <li>
     <a
-      href="https://www.usertesting.com/blog/site-navigation-tree-testing"
-      >https://www.usertesting.com/blog/site-navigation-tree-testing</a
-    >
+      href="https://help.usertesting.com/hc/en-us/articles/115003372331-What-is-Tree-Testing">https://help.usertesting.com/hc/en-us/articles/115003372331-What-is-Tree-Testing</a>
```

Comment:
The old link just redirects to their blog home page. I found a post from them about tree testing, but I am not sure if it applies. Should we remove this or find another link?

Reply:
Let's keep for now. Created a new issue to follow-up on this https://github.com/uswds/uswds-site/issues/2187.